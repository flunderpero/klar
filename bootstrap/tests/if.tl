let word = "Hello"

if word == "Hello" then
    print("OK: Matches if true")
end

if word == "empty if body" then 
end

if word == "words" then
    print("ERR: Matches if false???")
end

if word == "Hello" then
    print("OK: Matches if true again")
end
