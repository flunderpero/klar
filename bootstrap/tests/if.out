OK: Matches if true
OK: Matches if true again
